src/resolvers/query/platform-releases/index.unit.test.js

```diff
@@ -0,0 +1,38 @@
+import resolvers from "resolvers";
+import { graphql } from "graphql";
+import { makeExecutableSchema } from "graphql-tools";
+import { importSchema } from "graphql-import";
+
+// Import our application schema
+const schema = makeExecutableSchema({
+  typeDefs: importSchema("src/schema.graphql"),
+  resolvers
+});
+
+// Define our mutation
+const query = `
+  query updateAvailable {
+    updateAvailable {
+      version
+    }
+  }
+`;
+
+describe("updateAvailable", () => {
+  test("typical request is successful", async () => {
+    // Mock up some db functions.
+    const platformReleases = jest.fn();
+
+    // Construct db object for context.
+    const db = {
+      query: {
+        platformReleases
+      }
+    };
+
+    // Run the graphql mutation.
+    const res = await graphql(schema, query, null, { db });
+    expect(res.errors).toBeUndefined();
```

Comment:
```suggestion
    expect(res.errors).toBeUndefined();
    expect(rest.data).toHaveProperty("updateAvailable", null);
```

(That should test that it exists and is explicitly null which is what we want.)

Reply:
fixed.